week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,17 +45,21 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+function shakeBall() {
```

Comment:
This is a good start, but it looks like this one isn't quite done :) Can you try to finish it off? If you're stuck, we're always around on slack, or at homework club, to help out!